src/test/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/prisons/SupportedPrisonsServiceTest.kt

```diff
@@ -1,48 +1,112 @@
 package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.prisons
 
 import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
+import org.mockito.ArgumentMatchers.anyString
 import org.mockito.kotlin.any
+import org.mockito.kotlin.check
 import org.mockito.kotlin.given
 import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
 import org.mockito.kotlin.then
+import uk.gov.justice.digital.hmpps.prisonregister.model.PrisonDto
+import uk.gov.justice.digital.hmpps.prisonregister.model.PrisonTypeDto
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.client.PrisonRegisterClient
 import java.time.Instant
 
 class SupportedPrisonsServiceTest {
 
   private val supportedPrisonsRepository = mock<SupportedPrisonsRepository>()
-  private val supportedPrisonsService = SupportedPrisonsService(supportedPrisonsRepository)
+  private val prisonRegisterClient = mock<PrisonRegisterClient>()
+  private val supportedPrisonsService = SupportedPrisonsService(supportedPrisonsRepository, prisonRegisterClient)
 
-  @Test
-  fun `should return empty list if none`() {
-    given { supportedPrisonsRepository.findByActive(any()) }.willReturn(listOf())
+  @Nested
+  inner class FindSupportedPrisonCodes {
```

Comment:
Tests in this class already existed, just been moved to a class. GitHub should have recognised the change is mostly indentation but the diffs get worse by the week.